ops/demo/persistent/main.tf

```diff
@@ -50,17 +50,30 @@ module "db" {
   rg_location = local.rg_location
   rg_name     = local.rg_name
 
-  global_vault_id      = data.azurerm_key_vault.global.id
-  db_vault_id          = data.azurerm_key_vault.db_keys.id
-  db_encryption_key_id = data.azurerm_key_vault_key.db_encryption_key.id
-  subnet_id            = module.vnet.subnet_vm_id
-  dns_zone_id          = module.vnet.private_dns_zone_id
-  log_workspace_id     = module.monitoring.log_analytics_workspace_id
-  nophi_user_password  = random_password.random_nophi_password.result
+  global_vault_id = data.azurerm_key_vault.global.id
+  db_vault_id     = data.azurerm_key_vault.db_keys.id
+  // TODO: delete old_subnet_id when removing the old DB configuration
```

Comment:
Are the TODO items throughout this PR encapsulated in a ticket somewhere? Just want to make sure we don't forget them. If they're part of the original ticket, then please disregard. 😄

Reply:
Huh, I thought we had one but I can't find it. Created #3386 - good call out.

Reply:
Thank you! ^-^